.github/resources/minio/deployment.yaml

```diff
@@ -55,7 +55,7 @@ spec:
           ports:
             - containerPort: 9000
               protocol: TCP
-          imagePullPolicy: IfNotPresent
+          # imagePullPolicy: default - https://kubernetes.io/docs/concepts/containers/images/#imagepullpolicy-defaulting
```

Comment:
can we also revisit existing imagepolicies, for example apiserver has it set to Always https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/internal/apiserver/default/deployment.yaml.tmpl#L179 

we should be consistent across all pods

Reply:
Done.